### PR TITLE
Don't load imports from templates in the main doc.

### PR DESCRIPTION
**Pull request by cdata**

### CODE REVIEW

File: webcomponents-lite.js [hunk 1]
```diff
@@ -53,9 +53,9 @@
     'URL/URL.js',
     'WeakMap/WeakMap.js',
     'MutationObserver/MutationObserver.js',
+    'Template/Template.js',
```
Comment (sorvell) [inline webcomponents-lite.js]: I think we also need to do this ordering change in the build (`build.json`).

Comment (cdata) [inline webcomponents-lite.js]: Modified `WebComponents/build-lite.json`. `WebComponents/build.json` didn't seem to directly reference templates. Any suggestions?

Comment (cdata) [inline webcomponents-lite.js]: I chatted with Dan, and discovered that `build.json` pulls in ShadowDOM Polyfill, which has a separate `HTMLTemplate.js` implementation... so, I think `build-lite.json` is the only thing that needs to be modified?

Comment (sorvell) [inline webcomponents-lite.js]: That sounds right.

**Comment by azakus** [comment]
Looks like the tests timed out on loading `Template/runner.html`.

File: tests/Template/runner.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<title>Template Tests</title>
+<meta charset="utf-8">
```
Comment (azakus) [inline tests/Template/runner.html]: Needs to load WCT about here.

File: tests/Template/imports.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          window.externalScriptParsed1 = undefined;
+        });
+
+        test('links are not imported before stamping', function() {
+          assert.equal(window.remoteCurrentScriptExecuted, undefined);
+          assert.equal(window.externalScriptParsed1, undefined);
+        });
+
+        test('links are imported when stmped', function(done) {
+          document.addEventlistener('HTMLImportsLoaded', function() {
```
Comment (azakus) [inline tests/Template/imports.html]: `document.addEventListener`

File: tests/Template/runner.html (hunk 2 above)
Comment (cdata) [inline tests/Template/runner.html]: Done.

**Comment by cdata** [comment]
@azakus @sorvell I fixed up the tests a little. PTAL.

File: tests/Template/runner.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+<script src="../../bower_components/web-component-tester/browser.js"></script>
```
Comment (azakus) [inline tests/Template/runner.html]: replace `bower_components` with `..`

**Comment by azakus** [comment]
LGTM with nit

**Pull request merged**